code/controllers/subsystems/vote.dm

```diff
@@ -117,7 +117,9 @@ SUBSYSTEM_DEF(vote)
 	if(active_vote)
 		win_x = active_vote.win_x
 		win_y = active_vote.win_y
-	show_browser(user, interface(user.client),"window=vote;size=[win_x]x[win_y]")
+	var/datum/browser/popup = new(user, "vote", "Voting Panel", win_x, win_y)
+	popup.set_content(interface(user.client))
+	popup.open(use_onclose = TRUE)
```

Comment:
As this autoupdates, it's possible you should store the browser datums and update their html instead of recreating them.